internal/verifier/verifier.go

```diff
@@ -124,35 +124,11 @@ func chdir(dir string) {
 	}
 }
 
-// ProvenanceMetadataVerifier verifies provenances by comparing the
-// content of the provenance predicate against a given set of expected values.
-type ProvenanceMetadataVerifier struct {
```

Comment:
Why is the verifier removed?

Reply:
Oh! I see. Sorry. I missed that its `Verify` is effectively only calling `Verify` on `ProvenanceIRVerifier`.